src/puppetlabs/puppetdb/anonymizer.clj

```diff
@@ -31,11 +32,10 @@
    (contains? event "old_value")
    (contains? event "resource_type")))
 
-(defn report?
+(pls/defn-validated report? :- report/report-schema
```

Comment:
Nope I get it, it sucks but yeah, I see why we need this. Not that important, but it no longer returns false when it doesn't validate - I guess it just throws a validation exception instead. I figure this won't be a problem?